openshift/ci-operator/knative-images/cli-artifacts/Dockerfile

```diff
@@ -1,10 +1,18 @@
 # This is not generated Dockerfile, yet!
+ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
 ARG CLI_ARTIFACTS=registry.redhat.io/openshift-serverless-1/kn-cli-artifacts-rhel8:1.14
 
-FROM $CLI_ARTIFACTS
+FROM $CLI_ARTIFACTS as builder
+
+FROM $GO_RUNTIME
 
 ARG VERSION=knative-v1.15
 
+RUN mkdir -p /usr/share/kn/{linux_amd64,linux_arm64,linux_ppc64le,linux_s390x,macos_amd64,macos_arm64,windows}
+
+RUN mkdir -p /usr/share/kn
+
+COPY --from=builder /usr/share/kn /usr/share/kn
```

Comment:
Copying from previous stage to prevent label inheritance and related EC failures.